Pyrado/pyrado/policies/recurrent/rnn.py

```diff
@@ -42,7 +43,7 @@ class RNNPolicyBase(RecurrentPolicy):
     """ Base class for recurrent policies wrapping torch.nn.RNNBase subclasses """
 
     # Type of recurrent network. Is None in base class to force inheritance.
-    recurrent_network_type = None
+    recurrent_network_type: Optional[RNNBase] = None
```

Comment:
I disagree here, since for the subclasses, the type must be a subclass of `RNNBase`, and must not be None